liubai-backends/liubai-laf/cloud-functions/ai-shared.ts

```
@@ -2144,6 +2730,116 @@ export class TransformContent {
+
+    /**
+     *  2. extract text like:
+     *  {
+     *    "msgtype": "location",
+     *    "latitude": "30.168669",
+     *    "longitude": "120.134827",
+     *    "title": "浙江省杭州市滨江区浦沿街道滨文路868号",
+     *    "address": "浙江省杭州市滨江区浦沿街道滨文路868号"
+     *  }
+     * 
+     * 这个位置位于浙江省杭州市滨江区的浦沿街道......
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)